Новая стратегия - FAKE TLS MOD AUTO ALT

V3nilla opened the pull request:
Постараюсь объяснить вкратце. При прошлых FAKE TLS MOD стратегиях, увидел что некоторые сайты стали грузить очень долго, причем - без интерфейса, оказалось что это вызывала ошибка - ERR_SSL_PROTOCOL_ERROR, из-за чего ломалось все, и эта стратегия - эдакий комбайн из тех решений, которые существуют, и с ним данной ошибки - уже не возникает, да и мало-ли, может кому-то поможет

V3nilla committed:
New Strategy

V3nilla committed:
Merge branch 'Flowseal:main' into main

tarakan92 commented:
> Постараюсь объяснить вкратце. При прошлых FAKE TLS MOD стратегиях, увидел что некоторые сайты стали грузить очень долго, причем - без интерфейса, оказалось что это вызывала ошибка - ERR_SSL_PROTOCOL_ERROR, из-за чего ломалось все, и эта стратегия - эдакий комбайн из тех решений, которые существуют, и с ним данной ошибки - уже не возникает, да и мало-ли, может кому-то поможет

Помогло,дискорд заработал,но теперь не работает ютюб,точней не грузят видео

V3nilla committed:
Фикс обновления BattlEye 

Видимо РКН занюхал чуток запрещенного, да такого количества, что от этого аж пострадал CDN сервак BattlEye, вследствие чего - произошел отвал обновления античита и невозможности поиграть в мультиплеер

V3nilla committed:
Merge pull request #1 from V3nilla/battleye-update-fix

Фикс обновления BattlEye
